dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -423,7 +423,7 @@ export const Card = ({
 					</div>
 				</ContentWrapper>
 			</CardLayout>
-			{hasSublinks && noOfSublinks > 2 ? (
+			{hasSublinks && noOfSublinks >= 2 ? (
```

Comment:
I made the changes in this file to achieve parity with frontend for the last screenshot in the PR description. Is there a requirement that `supportingContent` should look like this on mobile?

![image](https://user-images.githubusercontent.com/19683595/185210060-09b368ae-58af-434a-8ea2-e5eabed52b2a.png)

Reply:
I don’t know, might be worth getting @HarryFischer’s input.

Reply:
I have verified with @HarryFischer and we are keeping the change to make full width.

Reply:
Just realised this change creates an unwanted diff: https://www.chromatic.com/test?appId=5dfcbf3012392c0020e7140b&id=62fd2da1519f2836f2cffc8a. I will revert this for this PR and will fix it in a following one with another bug regarding `SupportingContent` that I noticed.